api/internal/handler/ssl/ssl_test.go

```diff
@@ -19,146 +19,859 @@ package ssl
-        "labels":{
-            "build":"18",
-            "env":"dev",
-            "version":"v3"
-        }
-	}`
-	err = json.Unmarshal([]byte(reqBody), ssl2)
-	assert.Nil(t, err)
-	ctx.SetInput(ssl2)
-	ret, err = handler.Update(ctx)
-	assert.Nil(t, err)
-	// check the returned value
-	objRet, ok = ret.(*entity.SSL)
-	assert.True(t, ok)
-	assert.Equal(t, "1", objRet.ID)
-	assert.Equal(t, ssl2.Key, objRet.Key)
-	assert.Equal(t, ssl2.Cert, objRet.Cert)
-
-	//sleep
-	time.Sleep(time.Duration(100) * time.Millisecond)
-
-	// get and check
-	input = &GetInput{}
-	input.ID = "1"
-	ctx.SetInput(input)
-	ret, err = handler.Get(ctx)
-	stored = ret.(*entity.SSL)
-	assert.Nil(t, err)
-	assert.Equal(t, stored.ID, ssl.ID)
-	assert.Equal(t, stored.Labels["build"], "18")
-	assert.Equal(t, stored.Labels["env"], "dev")
-	assert.Equal(t, stored.Labels["version"], "v3")
-
-	//list
-	listInput := &ListInput{}
-	reqBody = `{"page_size": 1, "page": 1}`
-	err = json.Unmarshal([]byte(reqBody), listInput)
-	assert.Nil(t, err)
-	ctx.SetInput(listInput)
-	retPage, err := handler.List(ctx)
-	assert.Nil(t, err)
-	dataPage := retPage.(*store.ListOutput)
-	assert.Equal(t, len(dataPage.Rows), 1)
+func TestSSL_Get(t *testing.T) {
+	_cert, _ := getTestKeyCert(t)
+	tests := []struct {
+		caseDesc   string
+		giveInput  *GetInput
+		giveRet    *entity.SSL
+		giveErr    error
+		wantErr    error
+		wantGetKey string
+		wantRet    interface{}
+	}{
+		{
+			caseDesc:   "normal",
+			giveInput:  &GetInput{ID: "ssl1"},
+			wantGetKey: "ssl1",
+			giveRet: &entity.SSL{
+				BaseInfo: entity.BaseInfo{
+					ID: "ssl1",
+				},
+				Key:  "",
```

Comment:
we should return un-empty `Key` to check whether the value of `Key` can be omit in the api response

Reply:
Yes, I missed it, fixed. Thks.